primer/brand
# Remove size prop from InlineLink

## Summary

Fixes #446. InlineLinks will no longer take a size prop. This means that they will work in `Heading` and `Text` components which have different size scales.

## List of notable changes:


- **Updated** InlineLinks. They no longer take a `size` prop, but will instead always inherit font-size and line-height. This means that they work inside Heading as well as Text components

## What should reviewers focus on?

Are there any edge cases not considered? Are there any components that InlineLink could be placed in where it would not correctly inherit size?


## Steps to test:

You can view the InlineLink stories in Storybook, or check the diffs in the snapshots.

## Supporting resources (related issues, external links, etc):


## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">


![CleanShot 2023-10-23 at 10 19 18@2x](https://github.com/primer/brand/assets/12260694/3353f6f7-9e69-4389-943d-2119aff0cd2e)

 </td>
<td valign="top">


![CleanShot 2023-10-23 at 10 19 36@2x](https://github.com/primer/brand/assets/12260694/3181f772-60d2-41ba-8998-cd7af36a6995)

</td>
</tr>
</table>
